src/resources/postcss/tribe-common-admin/tec_baseline_admin/_header.pcss

```
@@ -25,29 +27,17 @@
 }
 
 .tec-admin-page__logo {
-	margin: 0;
-	padding: var(--tec-spacer-1);
-	vertical-align: middle;
-	width: var(--tec-spacer-7);
+	height: var(--tec-spacer-8);
+	margin-left: var(--tec-spacer-0);
+	padding: var(--tec-spacer-0);
+	width: var(--tec-spacer-8);
 }
 
-.tec-admin-page__header-title {
+h1.tec-admin-page__header-title {
```

Did we need the `h1` here? It's better if we don't add tags to the selectors.
```suggestion
.tec-admin-page__header-title {
```